### DISCUSSION TITLE
Issue with Copilot in Visual Studio 2022 not correcting the case of already-typed code.

### DISCUSSION THREAD

**Discussion by fishnet37222** [community/community#73147] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I've been using Copilot for a while now and I've noticed an annoying issue with its behavior in Visual Studio 2022.  When it's completing code that I've already started to type, it doesn't correct the case of the already-typed code the way Visual Studio's built-in IntelliSense does.

For example, when I'm writing C# code, the standard is to have all method and property names start with a capital letter.  With the built-in IntelliSense, I can start typing the property or method name in lower-case and when I press Tab or Enter to select the one I want, the case of what I've already typed will be modified to match the item I've selected.  It would be really helpful if Copilot behaved the same way.

**Comment by ayim** [comment]
Tracking this discussion here: https://developercommunity.visualstudio.com/t/Issue-with-Copilot-in-Visual-Studio-2022/10535379
